doc/liblouis.texi

```diff
@@ -1620,24 +1557,19 @@ are specified by the @code{correct} opcode. If there are no
 @code{correct} opcodes in a table, the pre-translation pass is not used.
 If any back-translation corrections have been specified then they are
 applied in a post-translation (i.e. the very last) pass.
-
-Note that like the @opcoderef{context} and multi-pass opcodes, the
-@code{correct} opcode must be preceded by @opcoderef{noback} or
-@opcoderef{nofor}.
-
```

Comment:
Why did you remove the note about `noback` and `nofor`? I thought that was pretty accurate :-)